### PR TITLE
tourist recommendations

### LINKED ISSUE #18: Create a new file for rarun1989 Plano,TX
- [x] Create a branch
- [x] Add commits
- [x] open a PR
- [ ] collaborate
- [ ] seek approval
- [ ] close

### PR DESCRIPTION
**Pull request by rarun1989**
closes #18

### CODE REVIEW

**Comment by epadmanaban** [comment]
👍

File: rarun1989-PLANO.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# PLANO, TX
```
Comment (epadmanaban) [inline rarun1989-PLANO.md:1]: how is the weather now?

Comment (rarun1989) [reply rarun1989-PLANO.md:1]: It's just fine here!

**Review by epadmanaban — APPROVED** [review]
looks great!, will visit Plano

**Pull request merged**